static/js/compose_fade.js

```diff
@@ -121,8 +121,8 @@ exports.would_receive_message = function (email) {
     }
 
     // PM, so check if the given email is in the recipients list.
-    var recipients = focused_recipient.reply_to.split(',');
-    return recipients.indexOf(email) !== -1;
+    var recipients = focused_recipient.reply_to.toLowerCase().split(',');
+    return recipients.indexOf(email.toLowerCase()) !== -1;
```

Comment:
Perhaps we should move this into a function in `util.js` similar to the ones below?  If we want to avoid new bugs like this showing up, we might want to have a standard that we're not doing raw email address comparisons scattered through the codebase.